docs/integration-test-helpers/masonry/MasonryContainer.js

```diff
@@ -199,7 +199,7 @@ export default class MasonryContainer extends Component<Props, State> {
     this.forceUpdate();
   };
 
-  loadItems: ({| force: boolean, from?: number, name?: string |}) => void = ({
+  customLoadItems: ({| force: boolean, from?: number, name?: string |}) => void = ({
```

Comment:
Why this change?

Reply:
Because loadItems doesn't have `name` nor `force` in the Flow type. 

![Screenshot by Dropbox Capture](https://user-images.githubusercontent.com/10593890/234612709-51968754-537b-4208-8fc7-79f8a425876b.png)